app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx

```diff
@@ -58,8 +58,12 @@ interface Props {
     primaryTagging: Section[] | undefined | null;
     secondaryTagging: Widget[] | undefined | null;
     controlled: boolean | undefined | null;
+    verifiedBy: {
+        id: string;
+    }[] | undefined | null;
     compact?: boolean;
     entryImage: Entry['image'] | undefined | null;
+    getEntries: () => void;
```

Comment:
This component should not know about "getEntries", the callback name should be different.
Eg: onControl, onVerify